src/components/Logo/Logo.tsx

```diff
@@ -1,9 +1,14 @@
-import React from "react";
-import PropTypes from "prop-types";
+import React, { ReactElement, FC } from "react";
 import cn from "classnames";
-import logo from "~/assets/images/stormkit-logo.svg";
+import logo from "@images/stormkit-logo.svg";
```

Comment:
Let's not use `@` for the import statement but rather the `~`. The `@` is also used with some node modules (such as `@babel` and it may create confusion. 

```suggestion
import logo from "~/assets/images/stormkit-logo.svg";
```